packages/runtime/runtime-utils/src/summarizerNode.ts

```diff
@@ -324,6 +331,11 @@ export class SummarizerNode implements ISummarizerNode {
                 // No base summary to reference
                 throw error;
             }
+            this.logger.logException({
```

Comment:
nit it would be great to have some telemetry even if we throw one line above

Reply:
i'm not sure where that throw goes and didn't want to double log